library/rtstartup/rsbegin.rs

```diff
@@ -35,6 +35,16 @@ pub unsafe fn drop_in_place<T: ?Sized>(to_drop: *mut T) {
     drop_in_place(to_drop);
 }
 
+// Frame unwind info registration
+//
+// Each module's image contains a frame unwind info section (usually
+// ".eh_frame").  When a module is loaded/unloaded into the process, the
+// unwinder must be informed about the location of this section in memory. The
+// methods of achieving that vary by the platform.  On some (e.g., Linux), the
+// unwinder can discover unwind info sections on its own (by dynamically
+// enumerating currently loaded modules via the dl_iterate_phdr() API and
+// finding their ".eh_frame" sections); Others, like Windows, require modules
+// to actively register their unwind info sections via unwinder API.
```

Comment:
These wrappers were originally added so that linking to libgcc_eh could be avoided if panic=abort was used. However this has become less relevant recently since we always link to libgcc in the linker flags. This PR now makes them completely redundant since std depends on the unwinding support in libgcc no matter which panic runtime is used.